models/commons/src/localized-string/transformers.ts

```diff
@@ -1,15 +1,19 @@
 import type { TLocalizedString, TLocalizedStringGraphql } from './types';
-
 import { Transformer } from '@commercetools-test-data/core';
 
+const isNil = (value: string | undefined) =>
+  value === undefined || value === null;
+
 const transformers = {
   graphql: Transformer<TLocalizedString, TLocalizedStringGraphql>('graphql', {
     replaceFields: ({ fields }) =>
-      Object.entries(fields).map(([locale, value]) => ({
-        locale,
-        value,
-        __typename: 'LocalizedString',
-      })),
+      Object.entries(fields)
+        .filter(([, value]) => !isNil(value))
```

Comment:
💯

Reply:
Hm. That's actually weird. I've already merged these changes two weeks ago: https://github.com/commercetools/test-data/pull/20/files

Was that revert?

Reply:
Nope. Master has that change already... :thinking:

Reply:
Is it some issue with master/main naming?